app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -149,8 +149,8 @@
     private TextToSpeech textToSpeech;
     private String[] array;
     private String timenow;
-    private int remcount1 ;
-    private String remquery;
+    private int reminder_query ;
```

Comment:
For variables name camelCases should be used snake_cases should be used for the string names snake_cases should be used.